mutex_schema.cql

```diff
@@ -0,0 +1,8 @@
+CREATE KEYSPACE IF NOT EXISTS cassandra_locks WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 1};
```

Comment:
I'll remove it anyway as it doesn't add anything to this; the table definition is the important part.